homeassistant/components/comfoconnect/const.py

```diff
@@ -0,0 +1,13 @@
+"""Constants for ComfoConnect component."""
+
+from typing import Final
+
+DOMAIN: Final = "comfoconnect"
+PLATFORM: Final = ["sensor", "fan"]
```

Comment:
```suggestion
PLATFORMS = ["sensor", "fan"]
```
Better to name it `PLATFORMS`, No need to mark all the consts with `Final`